css-four-column-layout/styles.css

```diff
@@ -0,0 +1,52 @@
+.container {
+  margin: auto;
+  max-width: 80%;
+  background-color: rgb(247 248 250);
+  padding-left: 50px;
+  padding-right: 50px;
+  padding-bottom: 13px;
```

Comment:
Only `margin` and `max-width` on the container, class, please shift these declarations to another class.